addon/components/o-s-s/currency-input.ts

```diff
@@ -31,14 +32,21 @@ export default class OSSCurrencyInput extends Component<OSSCurrencyInputArgs> {
     return this.selectedCurrency.symbol || '—';
   }
 
-  get selectedCurrency() {
-    if (isEmpty(this.args.currency)) {
+  get selectedCurrencyCode(): string {
+    return this.selectedCurrency.code || '—';
+  }
+
+  get selectedCurrency(): Currency {
+    if (this.emptyCurrency) {
       return this._currencies[0];
     }
-
     return this._currencies.find((currency: Currency) => currency.code === this.args.currency) || this._currencies[0];
   }
 
+  get emptyCurrency(): boolean {
+    return isEmpty(this.args.currency);
+  }
```

Comment:
Is this getter used in the template?

Reply:
+1 
Unless used in the template, the `isEmtpy(this.args.currency)` should just be added b ack to line 40.